### PR TITLE
fix: Add docs for running the metadata CLI for backup in k8s upgrade

### LINKED ISSUE #9444: [DOCS] Update Docs for Kubernetes Upgrades Section for 0.12.3 to 0.13.1

### PR DESCRIPTION
**Pull request by akash-jain-10**
### Describe your changes :

closes #9444

#
### Type of change :

- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by Vj-L — DISMISSED** [review]

File: openmetadata-docs/content/deployment/upgrade/kubernetes.md [hunk 1]
```diff
@@ -27,18 +27,24 @@ Below document is valid for upgrading Helm Charts from **0.12.x to 0.13.0**.
...
-  <InlineCallout
-    color="violet-70"
-    icon="luggage"
-    bold="Backup Metadata"
-    href="/deployment/backup-restore-metadata"
-  >
-    Learn how to back up MySQL data.
-  </InlineCallout>
-</InlineCalloutContainer>
+Before proceeding, pleae make sure you made a backup of your MySQL/Postgres DB behind OpenMetadata server. This step is extremely important for you to restore to your current state if any issues come up during the upgrade. It is recommended before upgrading you production instances.
```
Comment (pmbrull) [inline openmetadata-docs/content/deployment/upgrade/kubernetes.md:30]:
```suggestion
Before proceeding, please back up your MySQL/Postgres DB behind the OpenMetadata server. This step is crucial for restoring to your current state if any issues arise during the upgrade. It is recommended before upgrading your production instances.
```

File: openmetadata-docs/content/deployment/upgrade/kubernetes.md [hunk 2]
```diff
@@ -27,18 +27,24 @@ Below document is valid for upgrading Helm Charts from **0.12.x to 0.13.0**.
...
+
+Make sure you have a connectivity between your database (mysql / postgresql) with host machine where you will be running the below commands. If you are using the default database available with OpenMetadata Dependencies, make sure to port-forward the mysql service using `kubectl port-forward service/mysql 3306:3306`.
```
Comment (pmbrull) [inline openmetadata-docs/content/deployment/upgrade/kubernetes.md:32]:
```suggestion
Make sure you have connectivity between your database (MySQL / PostgreSQL) and the host machine where you will be running the below commands. If you are using the default database available with OpenMetadata Dependencies, make sure to port-forward the MySQL service using `kubectl port-forward service/mysql 3306:3306`.
```

File: openmetadata-docs/content/deployment/upgrade/kubernetes.md [hunk 3]
```diff
@@ -27,18 +27,24 @@ Below document is valid for upgrading Helm Charts from **0.12.x to 0.13.0**.
...
+
+1. `python -m venv venv`
+2. `source venv/bin/activate`
+3. `PIP_USER=false pip install openmetadata-ingestion==0.13.0.3`
```
Comment (pmbrull) [inline openmetadata-docs/content/deployment/upgrade/kubernetes.md:36]:
```suggestion
3. `pip install openmetadata-ingestion~=0.13.1`
```

Backup fixes will be there for 0.13.1

File: openmetadata-docs/content/deployment/upgrade/kubernetes.md [hunk 4]
```diff
@@ -27,18 +27,24 @@ Below document is valid for upgrading Helm Charts from **0.12.x to 0.13.0**.
...
-    href="/deployment/backup-restore-metadata"
-  >
-    Learn how to back up MySQL data.
-  </InlineCallout>
-</InlineCalloutContainer>
+Before proceeding, pleae make sure you made a backup of your MySQL/Postgres DB behind OpenMetadata server. This step is extremely important for you to restore to your current state if any issues come up during the upgrade. It is recommended before upgrading you production instances.
+
+Make sure you have a connectivity between your database (mysql / postgresql) with host machine where you will be running the below commands. If you are using the default database available with OpenMetadata Dependencies, make sure to port-forward the mysql service using `kubectl port-forward service/mysql 3306:3306`.
+
+1. `python -m venv venv`
```
Comment (pmbrull) [inline openmetadata-docs/content/deployment/upgrade/kubernetes.md:34]:
```suggestion
Then, follow the next steps to create a virtual environment and install the latest OpenMetadata Python package with the backup CLI:

1. `python -m venv venv`
```

**Review by pmbrull — CHANGES_REQUESTED** [review]

File: openmetadata-docs/content/deployment/upgrade/kubernetes.md (hunk 3 above)
Comment (akash-jain-10) [reply openmetadata-docs/content/deployment/upgrade/kubernetes.md:36]: In that cases, should it be 0.13.1.0 as we are following the pattern X.X.X.X ?

**Review by pmbrull — APPROVED** [review]

**Pull request merged**